lib/lightstep/span.rb

```diff
@@ -5,9 +5,12 @@ module LightStep
   #
   # See http://www.opentracing.io for more information.
   class Span
+    # Part of the OpenTracing API
+    attr_writer :operation_name
+
     # Internal use only
     # @private
-    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags
+    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags, :operation_name
```

Comment:
@ngauthier thanks, that makes sense!

![image](https://cloud.githubusercontent.com/assets/2251553/20018438/ec219962-a284-11e6-8fca-7e3978dafc78.png)